geniza/corpus/solr_queryset.py

```diff
@@ -18,6 +18,7 @@ class DocumentSolrQuerySet(AliasedSolrQuerySet):
         "notes": "notes_t",
         "needs_review": "needs_review_t",
         "pgpid": "pgpid_i",
+        "old_pgpids": "old_pgpid_i",
```

Comment:
should this be `old_pgpid_is` (not `old_pgpid_i`) to match the field def in `Document.index_data()`?